exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -452,10 +452,8 @@ impl fmt::Debug for JavaRuntimeProxy {
     }
 }
 
-impl From<JavaRuntimeProxy> for (u32, Box<dyn Runtime>) {
-    fn from(r: JavaRuntimeProxy) -> Self {
-        (JavaRuntimeProxy::RUNTIME_ID as u32, Box::new(r))
-    }
+impl WellKnownRuntime for JavaRuntimeProxy {
+    const ID: u32 = JavaRuntimeProxy::RUNTIME_ID;
```

Comment:
Why not `pub const ID: u32 = RuntimeIdentifier::Java as u32;`? And then get rid of `JavaRuntimeProxy::RUNTIME_ID`?

Reply:
Thanks, good idea